src/utils.jl

```diff
@@ -160,15 +160,13 @@ Returns a list of modules currently loaded by the system.  On non-Linux platform
 returns an empty list.
 """
 function get_loaded_modules()
-    try
-        filter!(split.(readlines("/proc/modules"))) do (name, size, count, deps, state, addr)
-            return state == "Live"
-        end
-    catch e
-        if isa(e, SystemError)
-            return Vector{String}[]
-        end
-        rethrow(e)
+    @static if !Sys.islinux()
+        return Vector{String}[]
+    end
+
+    !isfile("/proc/modules") && return Vector{SubString{String}}[]
```

Comment:
It looks like one path returns `Vector{String}[]` and the other returns `Vector{SubString{String}}[]`.  I think we can just safely return `Vector{String}[]` in both cases.

Reply:
https://github.com/staticfloat/Sandbox.jl/pull/80#issuecomment-1023787498